cli/genenotebook

```diff
@@ -385,5 +384,175 @@ Example:
   })
   .exitOverride(customExitOverride(setUserPassword));
 
+// Create new user account.
+const addUser = user
+  .command('add');
+
+addUser.description('Add a new user account')
+  .usage('[option]')
+  .arguments('<newUserName> <newPassword>')
+  .requiredOption('-u, --username <adminUsername>', 'GeneNoteBook admin username')
+  .requiredOption('-p, --password <adminPassword>', 'GeneNoteBook admin password')
+  .option('--port [port]', 'Port on which GeneNoteBook is running. Default: 3000')
+  .option('-e, --email [email]', 'User email')
+  .option('-f, --first-name <userFirstName>', 'User first name')
+  .option('-l, --last-name <userLastName>', 'User last name')
+  .option('-r, --role <userRole>', 'User role e.g: registered, user, curator, admin')
+  .action((userName, newPassword, {
+    username, password, port = 3000, ...opts
+  }) => {
+    if (!(userName && newPassword && username, password)) {
+      addUser.help();
+    }
+
+    const userRole = (opts.role === undefined ? 'registered' : opts.role);
+
+    new GeneNoteBookConnection({ username, password, port })
+      .call('addUser', {
+        userName,
+        newPassword,
+        emails: opts.email,
```

Comment:
check that email, firstname and lastname are valid/not empty (except if firstname and lastname are optional?)